Tests/MapboxMapsTests/Annotations/Generated/CircleAnnotationManagerTests.swift

```diff
@@ -493,6 +493,22 @@ final class CircleAnnotationManagerTests: XCTestCase, AnnotationInteractionDeleg
         expectation = nil
     }
 
+
+    func testHandleDragBeginIsDraggableFalse() throws {
```

Comment:
The test will be more stable if you specify `isDraggable` explicitly(default value may change).